packages/yuudachi/src/deploy.ts

```diff
@@ -37,49 +39,55 @@ import {
 	ReportMessageContextCommand,
 	ReportUserContextCommand,
 } from "./interactions/index.js";
+import { createI18next } from "./util/i18next.js";
 
 const rest = new REST({ version: "10" }).setToken(process.env.DISCORD_TOKEN!);
 
+await createI18next();
+
 try {
 	console.log("Start refreshing interaction (/) commands.");
 
-	await rest.put(Routes.applicationGuildCommands(process.env.DISCORD_CLIENT_ID!, process.env.DISCORD_GUILD_ID!), {
-		body: [
-			// Moderation
-			AntiRaidNukeCommand,
-			BanCommand,
-			CaseLookupCommand,
-			DurationCommand,
-			HistoryCommand,
-			KickCommand,
-			ReasonCommand,
-			ReferenceCommand,
-			RestrictCommand,
-			SoftbanCommand,
-			UnbanCommand,
-			WarnCommand,
-			LockdownCommand,
-			TimeoutCommand,
-			ClearCommand,
-			ReportUtilsCommand,
+	const moderation = [
+		AntiRaidNukeCommand,
+		BanCommand,
+		CaseLookupCommand,
+		DurationCommand,
+		HistoryCommand,
+		KickCommand,
+		ReasonCommand,
+		ReferenceCommand,
+		RestrictCommand,
+		SoftbanCommand,
+		UnbanCommand,
+		WarnCommand,
+		LockdownCommand,
+		TimeoutCommand,
+		ClearCommand,
+		ReportUtilsCommand,
+	].map((command) => formatCommandToLocalizations("moderation", command as any));
```

Comment:
`as any` ?

Reply:
The only way I could found to invalidate the `as const`

Reply:
If you know a better way, pls let me know